.github/workflows/integrations-common.yml

```diff
@@ -0,0 +1,105 @@
+name: Test Integrations
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: "The name of the integration to test"
+        required: true
+        type: string
+      working_directory:
+        description: "The working directory to run the tests in"
+        required: true
+        type: string
+      xcode_version:
+        description: "The Xcode version to use for the job"
+        required: true
+        type: string
+      test-destination-os:
+        description: "The test destination OS to use for the job"
+        required: true
+        type: string
+      device:
+        description: "The device to test on"
+        required: false
+        default: ""
+        type: string
+      platform:
+        description: "The platform to test on"
+        required: true
+        type: string
+      package-name:
+        description: "The package name to test (same as Package.swift name)"
+        required: true
+        type: string
+      macos_version:
+        description: "macOS version"
+        required: true
+        default: macos-26
+        type: string
+
+jobs:
```

Comment:
`h`: There are many similarities to `unit-test-common.yml`. Can't we reuse `unit-test-common.yml` instead of having plenty of code duplicated?